readium/lcp/src/main/java/org/readium/r2/lcp/service/LicensesService.kt

```diff
@@ -71,14 +69,20 @@ internal class LicensesService(
         licenseDocument: LicenseDocument,
         publicationFile: File
     ): Try<Unit, LcpError> {
+        licenseDocument.publicationLink.hash
```

Comment:
The `hash` might be missing in the LCP License Document, in which case we don't want to fail the injection.

> Reading Systems should verify the integrity of the downloaded Protected Publication, if a hash is provided.
> https://readium.org/lcp-specs/releases/lcp/latest.html#acquiring-the-publication